docs/ARCHITECTURE.md

```diff
@@ -323,7 +323,7 @@ fun ExampleScreen(
             text = state.exampleData,
             textAlign = TextAlign.Center,
             style = MaterialTheme.typography.headlineSmall,
-            color = MaterialTheme.colorScheme.onSurface,
+            color = BitwardenTheme.colorScheme.textColors.primary,
```

Comment:
May be overkill and definitely outside the scope of this PR. It may be a good idea to just call out the use of Material components and how we are creating this pattern of mapping them for some of those components like radio buttons/cards/etc. component somewhere in the Readme